toolchain/install/install_paths_test.cpp

```diff
@@ -16,6 +16,14 @@
 #include "tools/cpp/runfiles/runfiles.h"
 
 namespace Carbon {
+
+class InstallPathsTestPeer {
```

Comment:
(not for this PR, maybe for a follow-up)

Is "test peer" a well established term? I couldn't find it when searching, and it was new to me. And the term "peer" is specific enough that I somewhat expect it to have a fairly specific meaning.

If its not that established, I'm still perfectly happy with the pattern itself to allow testing -- but maybe `...TestHelper` as a name to be a little more generic? But as I said, not this PR, a follow-up to maybe rename both if folks like that.

Reply:
`Peer` specifically comes from https://abseil.io/tips/135

Reply:
Note, left to my own devinces, I'd just make InstallPathsTest itself the friend...

Reply:
(I'm merging, expecting we'll come up with some unified solution for InstallPathsTestPeer, TypedNodesTestPeer, and InstTestHelper)